src/geometry.rs

```diff
@@ -231,16 +231,44 @@ impl Size<f32> {
     }
 }
 
+impl Size<Option<f32>> {
+    /// TODO
+    #[must_use]
+    pub fn from_none() -> Self {
+        Size { width: None, height: None }
+    }
+
+    /// TODO
+    #[must_use]
+    pub fn from_some(width: f32, height: f32) -> Self {
```

Comment:
I definitely think the name could be improved here. Not immediately sure how.

Reply:
Hmm. This feels very similar to the `from_points` method below?

Reply:
Yeah, not fully sure what is a better naming. This constructs a `Size<Option<f32>>`, `from_points()` constructs a `Size<Dimension::Points(f32)>`

Perhaps the biggest downside is what to do when you need to create a method that create a `Size<Option<OtherType>>` in which case `from_some()` would be wholly ambiguous from naming alone.

Reply:
We could just call it `new`? This feels like the default constructor.